src/Transpiler.ts

```diff
@@ -714,8 +738,7 @@ export abstract class LuaTranspiler {
                 return this.transpileIdentifier(node as ts.Identifier);
             case ts.SyntaxKind.StringLiteral:
             case ts.SyntaxKind.NoSubstitutionTemplateLiteral:
-                const text = (node as ts.StringLiteral).text;
```

Comment:
Then this would just be something like `const text = this.escapeString((node as ts.StringLiteral).text);`